lib/katello/permissions/content_host_permissions.rb

```diff
@@ -20,7 +20,7 @@
   permission :edit_content_hosts,
              {
                'katello/api/v2/systems' => [:update, :refresh_subscriptions, :content_override, :add_subscriptions],
-               'katello/api/v2/system_packages' => [:install, :upgrade, :upgrade_all, :remove],
+               'katello/api/v2/host_packages' => [:install, :upgrade, :upgrade_all, :remove],
```

Comment:
This should actually be under :edit_hosts permissions, :edit_content_hosts will go away